src/sentry/static/sentry/app/views/performance/transactionSummary/userStats.tsx

```diff
@@ -54,9 +55,7 @@ function UserStats({eventView, totals, location, organization, transactionName}:
       );
     }
 
-    const apdexKey = `apdex_${threshold}`;
-    const formatter = getFieldRenderer(apdexKey, {[apdexKey]: 'number'});
-    apdex = formatter(totals, {organization, location});
+    apdex = totals[`apdex_${threshold}`].toFixed(4);
```

Comment:
Why is this using `.toFixed(4)` over the `formatter` here?

Reply:
invoking the formatter is overly verbose and the only thing it really does is `text-align: right` which we don't want here anyways.

Reply:
can we use [formatFloat](https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/utils/formatters.tsx#L180-L184) instead to be consistent with how we format numbers in (most of) the app.

Reply:
actually I think more stuff uses toFixed than formatFloat, but either way I want to kill formatFloat entirely because it's implemented inefficiently and functions very unexpectedly with large numbers or large `places`:

```
> formatFloat(1e11, 10)
1e-10
```

but i'm worried that something could possibly actually require this behavior.

Reply:
I mention using `formatFloat` because `.toFixed` has some gotchas to watch out for due to the precision of floating-point numbers see [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed#using_tofixed).

But mainly, the concern is to make sure we format `apdex` the same way in all parts of the application 🙂 (i.e. we don't format it with 4 decimal places here and 2 elsewhere)

Reply:
that's true, but i don't understand how the warning there is related to toFixed.

i did realize that the two aren't actually the same though: `toFixed` adds trailing zeroes, `formatFloat` doesn't. i kinda feel like trailing zeroes is more accurate, but it's better to be consistent.